webnn/clamp.https.any.js

```diff
@@ -0,0 +1,214 @@
+// META: title=test WebNN API clamp operation
+// META: global=window,dedicatedworker
+// META: script=./resources/utils.js
+// META: timeout=long
+
+'use strict';
+
+// https://webmachinelearning.github.io/webnn/#api-mlgraphbuilder-clamp
+
+const testClamp = async (operandType, syncFlag, inputValue, inputShape, expected, options) => {
+  const x = builder.input('x', {type: operandType, dimensions: inputShape});
+  const y = builder.clamp(x, options);
+  const TestTypedArray = TypedArrayDict[operandType];
+  const inputs = {'x': new TestTypedArray(inputValue)};
+  const outputs = {'y': new TestTypedArray(sizeOfShape(inputShape))};
+  let graph;
+
+  if (syncFlag) {
+    graph = builder.build({y});
+    context.compute(graph, inputs, outputs);
+  } else {
+    graph = await builder.buildAsync({y});
+    await context.computeAsync(graph, inputs, outputs);
+  }
+
+  const expectedData = expected instanceof Array ? new TestTypedArray(expected) : expected[operandType];
+  assert_array_approx_equals_ulp(outputs.y, expectedData, PrecisionMetrics.ULP[operandType].clamp, operandType);
+};
+
+// Input data are random float64 numbers between [-10, 10) generated by invoking getRandomArbitrary(-10, 10)
+// refering to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random#getting_a_random_number_between_two_values
+const inputData = [
+  -3.4356449350874696, -6.530945988411405,  -8.175760663838268, 2.0879641317522726,
+  -4.480150236948526,  -8.591504561715722,  5.071455429211573,  -6.618697702258771,
+  4.224577823136105,   6.450272349350044,   -8.799923845835664, -3.3445965406946643,
+  5.550524270215341,   1.2788677438688012,  9.333702625514768,  9.2261637863086,
+  -7.302720212371034,  1.7865902395032585,  5.564981581526375,  3.145101011211482,
+  -8.275078596251655,  -1.3557080837143296, 7.348269585030259,  -5.530012756488021,
+];
+// expected data by clamping input data with default options
+const expectedDefault = {
+  float32: [
```

Comment:
Could you please share how you convert the double precision results of webnn-baseline to float32?